src/proper_parenthetics.py

```diff
@@ -0,0 +1,54 @@
+"""Contains a Stack class and a script to evaluate the completion of pairs of
+opening and closing parenthesis in a string."""
+
+
+def get_parenthetics_status(input_string):
+    """Returns a number determining the status of open/closed paranthesis
+    in a string (1 for open, 0 for closed, -1 for broken)."""
+    stack = Stack()
+    for char in input_string[::-1]:
+        if char == ")" or char == "(":
+            stack.push(char)
+    count = 0
+    while stack.head is not None:
+        if count < 0:
+            return -1
+        val = stack.pop()
+        if val == "(":
+            count += 1
+        else:
+            count -= 1
```

Comment:
If I have a string like ")" then I pop that from the stack and now my count is `-1`. It also breaks the `while` loop, and since you have a bare `else` at the bottom, you would return `1` indicating that it's an "open" string instead of a "broken" one.